datashader/glyphs.py

```diff
@@ -2,18 +2,31 @@
 
 from toolz import memoize
 
+from .core import Expr
 from .utils import ngjit, isreal
 
 
-class Glyph(object):
+class Glyph(Expr):
+    """Base class for glyphs."""
     pass
 
 
 class Point(Glyph):
+    """A point, with center at ``x`` and ``y``.
```

Comment:
An idealized (zero area) point?

I.e., can you clarify that this is the mathematical definition of point, not a dot that has a specific area (which many plotting programs will call a point)?  Later we'll add a circle, but for now people may think this is meant to be a circle.